src/Altinn.Platform/Altinn.Platform.Events/Events/Migration/v0.03/01-setup-tables.sql

```diff
@@ -0,0 +1,41 @@
+-- Table: events.eventssubscription
+
+CREATE TABLE IF NOT EXISTS events.eventssubscription
```

Comment:
Will there be more subscription-tables or coud we just call this table events.subscription ?

Reply:
fixed